cypress/support/commands.js

```diff
@@ -113,22 +113,7 @@ Cypress.Commands.add('checkA11y', () => {
 
     cy.window({ log: false })
         .then(window => {
-            return window.axe.run(
-                {
-                    include: window.document,
-                    exclude: [['iframe']]
-                },
-                {
-                    rules: {
-                        // @TODO: Review and re-enable this
-                        'color-contrast': { enabled: false },
-                        // New ARIA complementary rule. Review and then re-enable
-                        'landmark-complementary-is-top-level': {
```

Comment:
I'd suppressed this rule when it was first introduced as I needed to review the recommendations first. The gist is our use of `<aside>` is broadly incorrect. Not a major accessibility issues but can cause some confusion. `<aside>` can be used outside `<main>` e.g. for related posts / additional content. But shouldn't be used inside `<main>` for "callout" style content.